data/sql/updates/pending_db_world/rev_1673668700960590200.sql

```diff
@@ -0,0 +1,244 @@
+DELETE FROM `creature` WHERE `map`=560 AND `id1` IN (2385, 2350, 2354, 2408) AND `guid` IN (83460,83461,83462,83463,83464,83465,83466,83467,83468,83469,83470,83471,83472,83473,83478,83479,83480,83481,83482,83483,83484,83485,83486,83487,83488,83489,83490,83491,83492,83493,83494,83495,83496,83517,83526,83527,83528,83529,83530,83531,83532,83533,83534,83535,83536,83537,83538,83539,83540,83541,83542,83543,83544,83545,83546,83547,83548,83549,83550,83551,83552,83553,83554,83555,83556,83557,83558,83559,83560,83561,83562,83563,83564,83565,83566,83567,83568,83569,83570,83571,83572,83573,83574,83575,83576,83577,83578,83579,83580,83581,83582,83583,83584,83585,83586,83587,83588,83589,83590,83591,83592,83593,83594,83595,83596,83597,83598,83599,83600,83601,83602,83603,83604,83605,83606,83607,83608,83609,83610,83611,83612,83613,83614,83615,83616,83617,83618,83619,83620,83621,83622,83623,83624,83625,83626,83654,83655,83656,83657,83658,83659,83660,83661,83662,83663,83664,83665,83666,83667,83732,83733,83734,83735,83736,83737,83738,83739,83740,83741,83742,83743,83744,83745,83746,83747,83748,83749,83750,83751,83752,83753,83754,83755,83756,83757,83758,83759,83760,83761,83762,83763,83764,83765,83766,83767,83768,83769,83770,83771,83772,83773,83774,83775,83776,83777,83778,83779,83780,83781,83782,83783,83784,83785,83786,83787,83788,83789,83790,83791,83792,83793,83794,83795,83796,83797,83798,83799,83800,83801,83802,83803,83804,83805,83806,83807,83808,83809,83810,83811,83812,83813,83814,83815,83816,83817,83818,83819,83820,83821,83822);
```

Comment:
why is DELETE FROM `creature` WHERE `map`=560 AND `id1` IN (2385, 2350, 2354, 2408) AND `guid` IN  repeated in both lines?

Reply:
One is for the old creatures, the other is for the INSERT ones.
I should've specified in comments, too lazy 😛

Reply:
Is this good to go for approval?

Reply:
ye